### DISCUSSION TITLE
Exclude multiple path from Filter

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5190] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/f72cf282-651b-4c6b-9aeb-adaba0cf3b63)

How can I specify another path along with the one defined in the above picture?

**Comment by DogmaDragon** [comment, marked as answer]
You will need to select **matches regex** and then add those multiple paths together with `|`. 
E.g. `/run/media/lamy/WD1/Exclusive/Secure/Video/1080|/run/media/lamy/WD1/Exclusive/Secure/Video/720` etc. 

You can test your regex against https://regex101.com (select Golang flavor).
